print.go fails to compile with tablewriter v1.0.7 due to removed constants and methods

**Description:**

When using [`github.com/olekukonko/tablewriter v1.0.7`](https://github.com/olekukonko/tablewriter) with this library, `print.go` fails to compile due to multiple undefined constants and missing method definitions. This seems to be caused by changes in `tablewriter` removing alignment constants like `ALIGN_LEFT` and several methods on `tablewriter.Table`.

Here’s the full list of compilation errors encountered with `parquet-go v0.25.0` and `tablewriter v1.0.7`:

```
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:228:31: undefined: tablewriter.ALIGN_LEFT
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:230:31: undefined: tablewriter.ALIGN_RIGHT
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:256:43: undefined: tablewriter.ALIGN_LEFT
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:271:5: tw.SetAutoFormatHeaders undefined (type *tablewriter.Table has no field or method SetAutoFormatHeaders)
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:272:5: tw.SetColumnAlignment undefined (type *tablewriter.Table has no field or method SetColumnAlignment)
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:273:5: tw.SetHeaderAlignment undefined (type *tablewriter.Table has no field or method SetHeaderAlignment)
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:273:36: undefined: tablewriter.ALIGN_LEFT
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:274:5: tw.SetFooterAlignment undefined (type *tablewriter.Table has no field or method SetFooterAlignment)
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:274:36: undefined: tablewriter.ALIGN_LEFT
../../go/pkg/mod/github.com/parquet-go/parquet-go@v0.25.0/print.go:275:5: tw.SetHeader undefined (type *tablewriter.Table has no field or method SetHeader)
```

This appears in `tablewriter` version `v1.0.7`. A potential solution could be to either:
- Pin a compatible version (prior to v1.0.7), or
- Refactor the code in `print.go` to work with the updated `tablewriter` API.

---

**Steps to Reproduce:**

1. Use the following in your `go.mod`:

   ```go
   require (
       github.com/parquet-go/parquet-go v0.25.0
       github.com/olekukonko/tablewriter v1.0.7
   )
   ```

2. Run `go build`

---

**Expected behavior:**

`print.go` should compile cleanly with `tablewriter v1.0.7`.

---

**Suggested Fix:**

- Update `print.go` to reflect API changes in `tablewriter v1.0.7`, or
- Pin the dependency to an earlier compatible version in `go.mod`.

--- a/go.mod
+++ b/go.mod
@@ -7,13 +7,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/klauspost/compress v1.17.9
-	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pierrec/lz4/v4 v4.1.21
 	golang.org/x/sys v0.21.0
 	google.golang.org/protobuf v1.34.2
-)
-
-require (
-	github.com/mattn/go-runewidth v0.0.15 // indirect
-	github.com/rivo/uniseg v0.4.7 // indirect
 )

--- a/go.tools.mod
+++ b/go.tools.mod
@@ -11,14 +11,12 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/klauspost/compress v1.18.0
-	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pierrec/lz4/v4 v4.1.22
 	golang.org/x/sys v0.30.0
 	google.golang.org/protobuf v1.36.5
 )
 
 require (
-	github.com/mattn/go-runewidth v0.0.9 // indirect
 	golang.org/x/mod v0.23.0 // indirect
 	golang.org/x/sync v0.11.0 // indirect
 	golang.org/x/tools v0.30.1-0.20250221230316-5055f70f240c // indirect

--- a/go.tools.sum
+++ b/go.tools.sum
@@ -8,10 +8,6 @@ github.com/hexops/gotextdiff v1.0.3 h1:gitA9+qJrrTCsiCl7+kh75nPqQt1cx4ZkudSTLoUq
 github.com/hexops/gotextdiff v1.0.3/go.mod h1:pSWU5MAI3yDq+fZBTazCSJysOMbxWL1BSow5/V2vxeg=
 github.com/klauspost/compress v1.18.0 h1:c/Cqfb0r+Yi+JtIEq73FWXVkRonBlf0CRNYc8Zttxdo=
 github.com/klauspost/compress v1.18.0/go.mod h1:2Pp+KzxcywXVXMr50+X0Q/Lsb43OQHYWRCY2AiWywWQ=
-github.com/mattn/go-runewidth v0.0.9 h1:Lm995f3rfxdpd6TSmuVCHVb/QhupuXlYr8sCI/QdE+0=
-github.com/mattn/go-runewidth v0.0.9/go.mod h1:H031xJmbD/WCDINGzjvQ9THkh0rPKHF+m2gUSrubnMI=
-github.com/olekukonko/tablewriter v0.0.5 h1:P2Ga83D34wi1o9J6Wh1mRuqd4mF/x/lgBS7N7AbDhec=
-github.com/olekukonko/tablewriter v0.0.5/go.mod h1:hPp6KlRPjbx+hW8ykQs1w3UBbZlj6HuIJcUGPhkA7kY=
 github.com/pierrec/lz4/v4 v4.1.22 h1:cKFw6uJDK+/gfw5BcDL0JL5aBsAFdsIT18eRtLj7VIU=
 github.com/pierrec/lz4/v4 v4.1.22/go.mod h1:gZWDp/Ze/IJXGXf23ltt2EXimqmTUXEy0GFuRQyBid4=
 github.com/xyproto/randomstring v1.0.5 h1:YtlWPoRdgMu3NZtP45drfy1GKoojuR7hmRcnhZqKjWU=

--- a/print.go
+++ b/print.go
@@ -6,8 +6,7 @@ import (
 	"io"
 	"strconv"
 	"strings"
-
-	"github.com/olekukonko/tablewriter"
+	"text/tabwriter"
 )
 
 func PrintSchema(w io.Writer, name string, node Node) error {
@@ -209,12 +208,11 @@ func sprint(name string, node Node) string {
 func PrintRowGroup(w io.Writer, rowGroup RowGroup) error {
 	schema := rowGroup.Schema()
 	pw := &printWriter{writer: w}
-	tw := tablewriter.NewWriter(pw)
+	tw := tabwriter.NewWriter(pw, 0, 0, 2, ' ', 0)
 
 	columns := schema.Columns()
 	header := make([]string, len(columns))
 	footer := make([]string, len(columns))
-	alignment := make([]int, len(columns))
 
 	for i, column := range columns {
 		leaf, _ := schema.Lookup(column...)
@@ -222,14 +220,25 @@ func PrintRowGroup(w io.Writer, rowGroup RowGroup) error {
 
 		header[i] = strings.Join(column, ".")
 		footer[i] = columnType.String()
-
-		switch columnType.Kind() {
-		case ByteArray:
-			alignment[i] = tablewriter.ALIGN_LEFT
-		default:
-			alignment[i] = tablewriter.ALIGN_RIGHT
-		}
 	}
+
+	// Print header
+	for i, h := range header {
+		if i > 0 {
+			pw.WriteString("\t")
+		}
+		pw.WriteString(h)
+	}
+	pw.WriteString("\n")
+
+	// Print separator line
+	for i := range header {
+		if i > 0 {
+			pw.WriteString("\t")
+		}
+		pw.WriteString(strings.Repeat("-", len(header[i])))
+	}
+	pw.WriteString("\n")
 
 	rowbuf := make([]Row, defaultRowBufferSize)
 	cells := make([]string, 0, len(columns))
@@ -253,11 +262,17 @@ func PrintRowGroup(w io.Writer, rowGroup RowGroup) error {
 					cells[columnIndex] = value.String()
 				} else {
 					cells[columnIndex] += "," + value.String()
-					alignment[columnIndex] = tablewriter.ALIGN_LEFT
 				}
 			}
 
-			tw.Append(cells)
+			// Print row
+			for i, cell := range cells {
+				if i > 0 {
+					tw.Write([]byte("\t"))
+				}
+				tw.Write([]byte(cell))
+			}
+			tw.Write([]byte("\n"))
 		}
 
 		if err != nil {
@@ -268,14 +283,23 @@ func PrintRowGroup(w io.Writer, rowGroup RowGroup) error {
 		}
 	}
 
-	tw.SetAutoFormatHeaders(false)
-	tw.SetColumnAlignment(alignment)
-	tw.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
-	tw.SetFooterAlignment(tablewriter.ALIGN_LEFT)
-	tw.SetHeader(header)
-	tw.SetFooter(footer)
-	tw.Render()
+	// Print footer
+	for i := range header {
+		if i > 0 {
+			pw.WriteString("\t")
+		}
+		pw.WriteString(strings.Repeat("-", len(header[i])))
+	}
+	pw.WriteString("\n")
+	for i, f := range footer {
+		if i > 0 {
+			pw.WriteString("\t")
+		}
+		pw.WriteString(f)
+	}
+	pw.WriteString("\n")
 
+	tw.Flush()
 	fmt.Fprintf(pw, "%d rows\n\n", rowGroup.NumRows())
 	return pw.err
 }

